### ISSUE TITLE
unfork Play in 2.12 build

### ISSUE THREAD

**Issue by SethTisue** [scala/community-builds#155]
```
  // TODO unfork: it replaces sys.error for unused dependency by println...
  playframework-ref            : "adriaanm/playframework.git#2.4.x"
```

**Comment by SethTisue** [comment]
to consider at the same time, is the following still needed?

```
  11: "old-typesafe-maven-releases: https://repo.typesafe.com/typesafe/maven-releases"
    // for play 2.3.9's netty-http-pipelining 1.1.2                                                                                                         
```

**Comment by SethTisue** [comment]
still forked after #299 but at least it's newer. @schmitch is working on it

**Comment by schmitch** [comment]
Guess if we merge that: https://github.com/playframework/playframework/pull/6593 it will work, however since sbt-rc isn't published i'm not sure if that will work?

Edit: if it's possible to override sbt-rc or if you disable the project it might already work. Or we might also add https://github.com/sbt/sbt-remote-control (client and actor-client)

**Comment by SethTisue** [comment]
> Or we might also add https://github.com/sbt/sbt-remote-control (client and actor-client)

let's try it: https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-integrate-community-build/665/consoleFull (404 at the moment, til the Jenkins queue clears)

**Comment by SethTisue** [comment]
looks like sbt-remote-control needs some updating for 2.12:

```
[sbt-remote-control] java.lang.RuntimeException: Unsupported scalaBinary version: 2.12.0-6df14e5-nightly
[sbt-remote-control]    at scala.sys.package$.error(package.scala:27)
[sbt-remote-control]    at TheBuild$$anonfun$makeClientProject$6.apply(build.scala:133)
[sbt-remote-control]    at TheBuild$$anonfun$makeClientProject$6.apply(build.scala:130)
[sbt-remote-control]    at sbt.EvaluateSettings$MixedNode.evaluate0(INode.scala:175)
```

not sure what you mean by "if it's possible to override sbt-rc" (dbuild builds everything itself from source) or "or if you disable the project" (in the Play build? if you push a commit showing what you mean, I can trigger a build on it)

**Comment by schmitch** [comment]
actually the problem with sbt-rc is that it doesn't use `%%` it uses the following version scheme: `"com.typesafe.sbtrc" % "client-2-12" % sbtRcVersion` however for running the `Play-Integration-Test` you won't need `sbt-remote-control` at all, thats why your branch worked with the 2.11 version.

Also there are only 4 projects that make use of it: `ForkRunProtocolProject`, `SbtForkRunProtocolProject`, `ForkRunProject` and `SbtForkRunPluginProject`. I guess disabling those would not even download the dependency (and they are not necessary for the tests at all, only for scripted tests I guess).

I also never did something with sbt-remote-control so I would first need to look into it (it's pretty rusty, but I have some time soon, starting at thursday I will probably take a look at it.)

**Comment by SethTisue** [comment]
ah, well we have `run-tests: false` anyway (not that fixing that wouldn't be good!), so maybe simply unforking will now work. I'll try it

sbt-remote-control looks pretty neglected, so I'm comfortable with not including it unless we must

**Comment by schmitch** [comment]
yeah we should probably create another ticket for the test fail. after https://github.com/scala/scala-dev/issues/229 they should work (hopefully)

**Comment by SethTisue** [comment]
sure, I opened https://github.com/scala/community-builds/issues/304 on it

**Comment by SethTisue** [comment]
yay, https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-integrate-community-build/667/consoleFull is green

**Comment by SethTisue** [comment]
fixed in #305 🎉
